app/Resources/views/invoice/login.html.twig

```diff
@@ -0,0 +1,56 @@
+{% extends 'base.html.twig' %}
+
+
+
+{% block body %}
+<div class="container">
+    <div class="row">
+        <div class="col-md-4 col-md-offset-4">
+            <div class="login-panel panel panel-default">
+                <div class="panel-heading">
+                    <h3 class="panel-title"><?php if (isset($_GET['successRegister'])): ?>You just registered successfully <?php endif ?>Please Sign In</h3>
```

Comment:
Poprawiłem ale jednej rzeczy nie rozumiem. 
{% if app.request.get('succesRegister',false) %}You just registered successfully{% else %}Please Sign In{% endif %}
Dlaczego succesRegister, false a nie true , widzę skutek ale według mojej logiki. IF coś jest spełnione wyświetl TO ELSE wyświetl TAMTO. Czy to true, false robi za negację "!"  ?